include/deal.II/grid/grid_tools.h

```diff
@@ -1032,6 +1032,34 @@ namespace GridTools
   ( const MeshType                                                                    &mesh,
     const std::function<bool (const typename MeshType::active_cell_iterator &)> &predicate );
 
+  /**
+   * Compute a vector of bounding boxes which encloses the space occupied by the cells which have the predicate
+   * property. If a single bounding box is required use GridTools::compute_bounding_box() .
+   *
+   * The algorithm loops over all cells at given level @p refinement_level and active cells at lower refinement
+   * levels, creating a Bounding Box for each of them. Then it merges "redundant" boxes i.e. those which can
+   * be expressed as a single one. In a last optional step all smaller boxes are merged with neighbours.
+   *
+   * The parameters to control the algorithm are:
+   * - @p predicate : the property of the cells to enclose e.g. IteratorFilters::LocallyOwnedCell .
+   *  The predicate is tested only on active cells.
+   * - @p refinement_level : it defines the level at which the initial Bounding Boxes are created. The refinement
+   *  should be set to a low refinement level. If @p refinement_level is higher than the number of levels of the
+   *  triangulation an exception is thrown.
+   * - @p max_boxes : the maximum number of bounding boxes to compute. If more are created the smaller ones are
+   *  merged with neighbours. By default all boxes are kept after merging and erasing the reduntant ones.
```

Comment:
I don't know what this means: "all boxes are kept after merging and erasing". Do you erase them, or do you keep them?

reduntant -> redundant